web-common/src/features/file-explorer/FileExplorer.svelte

```diff
@@ -103,7 +103,7 @@
   ) {
     const isCurrentFile =
       removeLeadingSlash(fromDragData.filePath) ===
-      removeLeadingSlash($page.params.file);
+      removeLeadingSlash($page.params.file ?? "");
```

Comment:
**Security Issue**: Potential security risk by using nullish coalescing operator without proper validation, leading to unintended behavior or JavaScript injection. <br> **Fix**: Ensure the value of '$page.params.file' is sanitized and validated before use to prevent JavaScript injection vulnerabilities. <br> **Code Suggestion**: 
 ```
 -      removeLeadingSlash($page.params.file);
 +      removeLeadingSlash(sanitizeInput($page.params.file ?? ""));
 ```

Reply:
Not relavant here